authorization-service/.babelrs

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Since we are working with esbuild, I think we can safely remove all babel related files and configurations (and packages)

Reply:
done